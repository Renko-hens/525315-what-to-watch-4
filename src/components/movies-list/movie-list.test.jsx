import React from "react";
import renderer from "react-test-renderer";
import ListMovies from "./movie-list.jsx";

const movies = [
  {
    title: `Пираты карбинского моря`,
    poster: {
      src: `img/johnny-english.jpg`,
      alt: `Пираты карбинского моря`,
    },
    preview: {
      src: `http://techslides.com/demos/sample-videos/small.webm`,
    },
    addressPage: `movie-page.html`,
  },
  {
    title: `Преступление Гринлевальда`,
    poster: {
      src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
      alt: `Преступление Гринлевальда`,
    },
    preview: {
      src: `http://techslides.com/demos/sample-videos/small.webm`,
    },
    addressPage: `movie-page.html`,
  },
];

describe(`Render ListCards Components`, () => {
  it(`Should ListCards Component`, () => {
    const tree = renderer.create(
        <ListMovies
          movies={movies}
          renderCard={() => {}}
          onSelectMovieCardClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
