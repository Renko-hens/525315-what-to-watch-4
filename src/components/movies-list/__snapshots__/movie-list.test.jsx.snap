// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ListCards Components Should ListCards Component 1`] = `
<div
  className="catalog__movies-list"
/>
`;
